tests/src/test/java/org/eclipse/hono/tests/registry/CredentialsAmqpIT.java

```diff
@@ -26,17 +26,19 @@
 import java.util.Base64;
 import java.util.concurrent.TimeUnit;
 
+import io.vertx.core.Future;
 import org.eclipse.hono.client.CredentialsClient;
 import org.eclipse.hono.client.HonoClient;
 import org.eclipse.hono.client.ServiceInvocationException;
 import org.eclipse.hono.tests.IntegrationTestSupport;
 import org.eclipse.hono.util.Constants;
 import org.eclipse.hono.util.CredentialsConstants;
 import org.eclipse.hono.util.CredentialsObject;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
-import org.junit.Rule;
+import org.junit.After;
 import org.junit.Test;
+import org.junit.Rule;
+import org.junit.BeforeClass;
+import org.junit.AfterClass;
```

Comment:
this does not comply with the import ordering we use in Hono. Please configure your IDE to use the order defined in `eclipse/hono.importorder`